services/autoscaling/Dockerfile

```diff
@@ -32,11 +33,10 @@ RUN --mount=type=cache,target=/var/cache/apt,mode=0755,sharing=private \
   && apt-get update \
   && apt-get install -y --no-install-recommends \
   # only the cli is needed and we remove the unnecessary stuff again
-  docker-ce-cli \
+  docker-ce-cli=${DOCKER_APT_VERSION} \
   && apt-get remove -y\
   ca-certificates \
   curl \
-  gnupg \
   lsb-release \
```

Comment:
why this change?

Reply:
it's already installed above, no need to reinstall it

Reply:
this is about removing it